added some discussion of HCL parsing context (#9501)
hashicorp/nomad: website/pages/docs/job-specification/hcl2/index.mdx

guide](/guides/hcl).
--->

## HCL Parsing Context

> It is important to acknowledge that the Nomad API uses JSON to represent Nomad jobs.

Reviewer: Let's add a link to the api docs and try to cut straight to the point.

```suggestion
The [Nomad API uses JSON][jobs-api], not HCL, to represent Nomad jobs.

[jobs-api]: /api-docs/jobs
```
